apps/pie-docs/src/_includes/footer.njk

```diff
@@ -1,8 +1,43 @@
+{% set allAboutPiePages = collections.all | eleventyNavigation("All about PIE") %}
+{% set designersPages = collections.all | eleventyNavigation("designers") %}
+{% set engineersPages = collections.all | eleventyNavigation("engineers") %}
+{% set supportPages = collections.all | eleventyNavigation("Support") %}
+{% set footerNavItems = [allAboutPiePages, designersPages, engineersPages, supportPages] %}
+{% set jetLogoSrc = '/assets/img/jet-logo.svg' | url %}
+{% set jetLogoSrcNarrow = '/assets/img/jet-logo-narrow.svg' | url %}
+
 <footer class="c-footer" data-test-id="site_footer">
-    <div class="c-footer-content c-content-container">
+    <div class='c-content-container'>
         <div class="c-footer-logo">
-            {% include "pie-logo.njk" %}
-        </div>
-        <a href="https://www.just-eat.co.uk/info/privacy-policy">Privacy policy</a>
-    </div>
-</footer>
+            <picture>
+                <source media="(max-width: 600px)" srcset={{ jetLogoSrcNarrow }}>
+                    <img src={{ jetLogoSrc }} alt="Just Eat Takeaway- Logo"></picture>
+                    <span class='c-footer-logo-divider' aria-hidden></span>
+                    {% include "pie-logo.njk" %}
+                </div>
+                <main class='c-footer-main'>
+                    {% for section in footerNavItems %}
```

Comment:
@raoufswe this looks great, I think we could probably add some navigation tests here like we do for the sidebar. Might be worth chatting with @siggerzz about?

Reply:
will reach out to @siggerzz about this 💯